design/ceph/ceph-exporter.md

```diff
@@ -0,0 +1,121 @@
+# Ceph exporter
+
+### Goals
+Use Ceph exporter as the new source of metrics based on performance counters coming from every Ceph daemon.
+
+## Use case
+Ceph exporter was created to avoid performance problems at scale obtaining metrics in clusters with high number of daemons (mainly OSDs).
+The Prometheus manager module was not enough performant to obtain the performance counters from all the daemons along the whole Ceph cluster, so the solution for avoid this problem in big clusters is to have a new daemon collecting all the performance counters from all the daemons in each host, and expose them as standard prometheus metrics at /metrics endpoint for prometheus to scrape each of those endpoints.
+### Why do we need this daemon in Rook
+Although probably the Rook clusters usually do not have a huge number of daemons, and therefore, probably the main problem solved by the Ceph Exporter(scalability) is not going to be present, with the Ceph exporter, the source of metrics changes in the Ceph clusters.
+After introducing Ceph Exporter in Ceph cluster we are going to have two main sources of metrics:
+- Prometheus manager module: It is responsible for exposing all metrics other than ceph daemons performance counters.
```

Comment:
If the exporter is not running, will the ceph daemons still report to this module?

Reply:
> If the exporter is not running, will the ceph daemons still report to this module?

https://github.com/rook/rook/pull/11360#discussion_r1035593705